frontend/src/pages/Admin/Components/Faq/Faq.jsx

```diff
@@ -40,6 +40,19 @@ export function Faq(props) {
             </div>
           </div>
         </div>
+        <div className={style["card-item"]}>
```

Comment:
as you are adding this manage Q&A card here change the scetion name from `FAQs` to `FAQs and Q&As`